project_template/webpack.config.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
   output: {
     path: out,
     filename: "[name].[hash].js",
-    publicPath: "./"
+    publicPath: "/"
```

Comment:
Hm ok but then we still need to make sure that this doesn't break Cordova, can you double check that? (`yarn run build && open dist/index.html` should be enough)